backends/basic/local.js

```diff
@@ -35,11 +35,7 @@ export default class Local extends Backend {
 		return {type: exists ? "update" : "create"};
 	}
 
-	static parseURL (source) {
-		let ret = super.parseURL(source);
-		ret.key = ret.url.pathname;
-		return ret;
-	}
-
-	static protocol = "local:";
+	static urls = [
+		{protocol: "local:", pathname: ":key(.+)"},
```

Comment:
Even though `protocol: "local:"` works fine, the actual result after instantiating a new `URLPattern` object for this property is `local` (no semicolon). Should we be consistent with the URL Pattern API, or is it OK to go with `local:`?

Reply:
Oh if `protocol: "local"` works, let's use that

Reply:
Will do.